source/features/rgh-feature-descriptions.tsx

```diff
@@ -135,8 +135,8 @@ async function add(infoBanner: HTMLElement): Promise<void> {
 	// This ID exists whether the feature is documented or not
 	const id = meta?.id ?? filename;
 
-	addDescription(infoBanner, id, meta);
 	await addDisabledBanner(infoBanner, id);
+	addDescription(infoBanner, id, meta);
```

Comment:
This change makes it dependent on the disabled banner code, we don't have to do that. Check out the previous solution in

- https://github.com/refined-github/refined-github/pull/6345